Sources/LinkPreview/LinkPreviewURLRequest.swift

```diff
@@ -14,11 +14,13 @@ struct LinkPreviewURLRequest {
         case fileURL(URL, String)
     }
     var request: HTTPClientRequest
-    let url: URL
+	let url: URL
+	let requestTimeout: Int64
 
-    init(url: URL) {
+	init(url: URL, timeout requestTimeout: Int64 = 5) {
```

Comment:
Given that this type is internal, I think we don't need to default the parameter here (I wouldn't want to accidentally forget to propagate the client-specified value)